our_site/planner/templates/planner/event_modify_invitees.html

```diff
@@ -1,49 +1,77 @@
-{% extends 'base.html' %}
-
-{% block content %}
-
-    <div class="column">
-        {% if messages %}
-            <ul class="messages">
-                {% for message in messages %}
-                    <li{% if message.tags %} class="{{ message.tags }}"{% endif %}>{{ message }}</li>
-                {% endfor %}
-            </ul>
-        {% endif %}
-
-        <div>
-             <form class="is-inline-block is-size-5" method="post" action="">
-                {% csrf_token %}
-                <input class="is-inline-block is-size-5" type="submit" value="Invite User" name="add">
-                <div class="is-inline-block">
-                    {{ invite_form.as_div }}
-                </div>
-            </form>
-
-            <form action="{% url "planner:dashboard" %}" class="is-inline-block is-pulled-right">
-                <input class="is-size-5" type="submit" value="Back to Dashboard">
-            </form>
-            <form action="{{ event_edit_page }}" class="is-inline-block is-pulled-right">
-                <input class="is-size-5" type="submit" value="Back to Event Info">
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <!-- Include the Bulma CSS framework for styling -->
+    <link rel="stylesheet"
+          href="https://cdn.jsdelivr.net/npm/bulma@0.9.3/css/bulma.min.css">
+    <title>Planner</title>
+</head>
+<body>
+    <section class="hero is-small mb-4" style="background-color: #eeb01e">
+        <div class="hero-body">
+            <h1 class="title is-1" style="color: #7f0000">Add Invitees!</h1>
+            <p class="has-text-right subtitle is-4" style="color: #7f0000">
+                {{ user }} is signed in
+            </p>
+            <form class="has-text-right" action="{% url 'login' %}">
+                <input class="button is-default has-text-warning" style="background-color: #7f0000" type="submit" value="Logout" />
             </form>
         </div>
+    </section>
 
-        <hr>
-        <header class="is-size-3 has-text-centered">Current Invitations:</header>
+    <div class="container">
 
-         {% for invitee in invitees %}
-            <div class="box">
-                <form class="is-inline-block" method="post" action="">
+        {% block content %}
```

Comment:
Is there a reason we have block content in here if there is no longer any inheritance?

Reply:
I guess this is a question about all the files which no longer use inheritance but still have block content?

Reply:
got rid of them!